text.cpp

```diff
@@ -396,5 +396,70 @@ end)";
+(Tidak bekerja dalam mode administrator)";
+	USE_JP_LOCALE = u8"Gunakan locale jepang?";
+	DEFAULT_CODEPAGE = u8"Codepage standar";
+	FLUSH_DELAY = u8"Flush Delay";
+	MAX_BUFFER_SIZE = u8"Max Buffer Size";
+	CONSOLE = L"Konsol";
+	CLIPBOARD = L"Papan clipboard";
+	ABOUT = L"Textractor " ARCH L" v" VERSION LR"( dibuat oleh saya: Artikash (email: akashmozumdar@gmail.com)
+Halaman project: https://github.com/Artikash/Textractor
+Video tutorial : https://www.youtube.com/watch?v=eecEOacF6mw
+Tolong hubungi saya jika kamu memiliki masalah terkait masalah, permintaan fitur, atau pertanyaan terkait Textractor
+Kamu dapat melakukannya lewat halaman utama project (bagian issues) atau lewat email
+Source code tersedia dibawah lisensi GPLv3 di halaman utama project
+Saya sedang mencari pekerjaan baru : email saya jika kamu mengenal orang yang dapat memperkerjakan software engineer Amerika
+Jika kamu menyukai project ini, tolong sebarluaskan project ini :))";
+	UPDATE_AVAILABLE = L"Pembaharuan tersedia: pembaharuan dapat di unduh di https://github.com/Artikash/Textractor/releases";
+	ALREADY_INJECTED = L"Textractor: sudah ditempelkan";
+	INJECT_FAILED = L"Textractor: menempelkan gagal";
+	LAUNCH_FAILED = L"Textractor: game tidak dapat dijalankan";
+	INVALID_CODE = L"Textractor: kode tidak sesuai";
+	INVALID_CODEPAGE = L"Textractor: tidak dapat mengkonversi teks (Codepage tidak sesuai?)";
+	PIPE_CONNECTED = u8"Textractor: pipe tersambung";
+	INSERTING_HOOK = u8"Textractor: memasukkan hook: %s";
+	REMOVING_HOOK = u8"Textractor: menghapus hook: %s";
+	HOOK_FAILED = u8"Textractor: gagal memasukkan hook";
+	TOO_MANY_HOOKS = u8"Textractor: terlalu banyak hook: tidak dapat memasukkan hook";
+	NOT_ENOUGH_TEXT = u8"Textractor: tidak cukup teks untuk melakukan pencarian secara akurat";
+	FUNC_MISSING = u8"Textractor: tidak ada fungsi";
+	MODULE_MISSING = u8"Textractor: tidak ada modul";
+	GARBAGE_MEMORY = u8"Textractor: memory terus berganti, tidak dapat dibaca";
+	SEND_ERROR = u8"Textractor: Send ERROR (kemungkinan H-Code salah)";
+	READ_ERROR = u8"Textractor: Reader ERROR (Kemungkinan R-Code salah)";
+	HIJACK_ERROR = u8"Textractor: Hijack ERROR";
+	COULD_NOT_FIND = u8"Textractor: tidak dapat menemukan teks";
+	SELECT_LANGUAGE = u8"Pilih bahasa";
+	BING_PROMPT = u8"Bahasa apakah yang Bing harus terjemahkan?";
+	GOOGLE_PROMPT = u8"Bahasa apakah yang Google harus terjemahkan?";
+	TOO_MANY_TRANS_REQUESTS = L"Terlalu banyak permintaan terjemahan: menolak untuk menerjemahkan";
+	TRANSLATION_ERROR = L"Terjadi kesalahan ketika menerjemahkan";
+	EXTRA_WINDOW_INFO = u8R"(Klik kanan untuk merubah pengaturan
+Klik dan tarik pinggiran jendela untuk memindahkan, atau sudut kanan bawah untuk mengatur ukuran jendela)";
+	BG_COLOR = u8"Warna latar";
+	TEXT_COLOR = u8"Warna teks";
+	FONT_SIZE = u8"Ukuran teks";
+	TOPMOST = u8"Selalu berada di atas";
+	REGEX_FILTER = u8"Filter regex";
+	INVALID_REGEX = u8"Regex tidak sesuai";
+	CURRENT_FILTER = u8": Regex yang digunakan sekarang";
```

Comment:
This won't work; this results in Regex Filter displaying, for example, `: Regex yang digunakan sekarang[A-Z]` when the current filter is `[A-Z]`. It's just a concatenation, is there any way indonesian can reverse this? Otherwise I'll change it to a format string.